### PR TITLE
RDM-4184 Remove case history tab as it is provided by flexible contro…

### PR DESCRIPTION
**Pull request by andrewfolga**
…l now.
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-4184

### Change description ###
Remove case history tab as it is provided by flexible control now.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by gbenadikar** [comment]
LGTM

File: src/shared/components/case-viewer/case-viewer.component.spec.ts [hunk 1]
```diff
@@ -38,9 +38,12 @@ class TabsComponent { }
   selector: 'cut-tab',
   template: '<ng-content></ng-content>'
 })
-class TabComponent { }
+class TabComponent {
+  @Input()
+  selected: boolean;
+}
 
-describe('CaseViewerComponent', () => {
```
Comment (andrewfolga) [inline src/shared/components/case-viewer/case-viewer.component.spec.ts]: Run all tests.

File: src/shared/components/case-viewer/case-viewer.component.spec.ts [hunk 2]
```diff
@@ -603,15 +618,6 @@ describe('CaseViewerComponent', () => {
     expect(orderService.sort).toHaveBeenCalledWith(FIELDS);
   });
 
-  it('should render the event log component in the event log tab', () => {
```
Comment (andrewfolga) [inline src/shared/components/case-viewer/case-viewer.component.spec.ts]: This is no longer on the case viewer template level. It's tested down in the case-history-viewer level.

**Comment by veeresha-hmcts** [comment]
Verified and ready for merge.

**Pull request merged**